index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div>
+      <h1>Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+    </div>
+    <section class="container">
+      <picture>
+        <img src="./Assets/tribute-page-main-image.jpg" alt="" />
```

Comment:
You didn't add the alt description to the image

Reply:
- [ ] I recommend following my previous comments because they will help in improving your communication skills :100:.